lib/hammer_cli/defaults.rb

```diff
@@ -0,0 +1,143 @@
+require 'hammer_cli'
+require 'yaml'
+module HammerCLI
+
+  class DefaultsCommand < HammerCLI::AbstractCommand
+
+    class ListDefaultsCommand < HammerCLI::DefaultsCommand
+      command_name 'list'
+
+      desc _('List all the defaults parameters')
```

Comment:
'defaults' typo?. Should probably be either 'List all default parameters' or 'List all the defaults'.